server/src/unifyfs_request_manager.c

```diff
@@ -1381,6 +1413,12 @@ void* request_manager_thread(void* arg)
         thrd_ctrl->waiting_for_work = 0;
         RM_UNLOCK(thrd_ctrl);
 
+        rc = rm_heartbeat(thrd_ctrl);
```

Comment:
It's hard for me to tell from the code, but assuming the resource manager thread has no work to do, how often are we pinging the client?

The comments above suggest the ``pthread_cond`` times out after 10ms.

Reply:
The code in `rm_heartbeat()` makes it so it only does the RPC once every 30 seconds.

Reply:
Ok, thanks.  Now I see it.